var/spack/repos/builtin/packages/fujitsu-ssl2/package.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class FujitsuSsl2(Package):
+    """Fujitsu SSL2 (Scientific Subroutine Library II) is Math library
+    for Fujitsu compiler.
+    Fujitsu SSL2 implementation only for Fujitsu compiler.
+    Fujitsu SSL2 provides the function of blas, lapack and scalapack.
+    """
+
+    homepage = "https://www.fujitsu.com/us/"
+
```

Comment:
You'll want to add the following to avoid `fetch` errors since there is no `url`:
```suggestion
    has_code = False
```